geomesa-accumulo-bundle/geomesa-accumulo-processors/src/test/scala/org/geomesa/nifi/processors/accumulo/PutGeoMesaAccumuloTest.scala

```diff
@@ -397,4 +399,71 @@ class PutGeoMesaAccumuloTest extends LazyLogging {
       ds.dispose()
     }
   }
+
+  @Test
+  def testUpdateIngest(): Unit = {
+    val catalog = s"${root}UpdateIngest"
+    val runner = TestRunners.newTestRunner(new PutGeoMesaAccumulo())
+
+    def checkResults(ids: Seq[String], names: Seq[String], dates: Seq[Date], geoms: Seq[String]): Unit = {
+      val ds = DataStoreFinder.getDataStore((dsParams + (AccumuloDataStoreParams.CatalogParam.key -> catalog)).asJava)
+      Assert.assertNotNull(ds)
+      try {
+        val sft = ds.getSchema("example")
+        Assert.assertNotNull(sft)
+        val features = SelfClosingIterator(ds.getFeatureSource("example").getFeatures.features()).toList.sortBy(_.getID)
+        logger.debug(features.mkString(";"))
+        Assert.assertEquals(3, features.length)
+        Assert.assertEquals(ids, features.map(_.getID))
+        Assert.assertEquals(names, features.map(_.getAttribute("name")))
+        Assert.assertEquals(dates, features.map(_.getAttribute("dtg")))
+        Assert.assertEquals(geoms, features.map(_.getAttribute("geom").toString))
+      } finally {
+        ds.dispose()
+      }
+    }
+
+    val df = new SimpleDateFormat("yyyy-MM-dd")
+    try {
+      dsParams.foreach { case (k, v) => runner.setProperty(k, v) }
+      runner.setProperty(AccumuloDataStoreParams.CatalogParam.key, catalog)
+      runner.setProperty(FeatureTypeProcessor.Properties.SftNameKey, "example")
+      runner.setProperty(ConverterIngestProcessor.ConverterNameKey, "example-csv")
+      runner.enqueue(getClass.getClassLoader.getResourceAsStream("example.csv"))
+      runner.run()
+      runner.assertTransferCount(Relationships.SuccessRelationship, 1)
+      runner.assertTransferCount(Relationships.FailureRelationship, 0)
+      checkResults(
+        Seq("23623", "26236", "3233"),
+        Seq("Harry", "Hermione", "Severus"),
+        Seq("2015-05-06", "2015-06-07", "2015-10-23").map(df.parse),
+        Seq("POINT (-100.2365 23)", "POINT (40.232 -53.2356)", "POINT (3 -62.23)")
+      )
+      runner.setProperty(AbstractDataStoreProcessor.Properties.WriteMode, FeatureWriters.Modify)
```

Comment:
In the docs PR (https://github.com/locationtech/geomesa/pull/2649/files), can you say a little more about how one may use a processor for writes and updates?  Or were you thinking that one would use a processor for one or the other?

Reply:
yeah, you would set the processor for either writes or updates, but not both at once (you would configure 2 different processors instead)